tutorials/00-introduction/00_introduction.jmd

```diff
@@ -106,31 +114,51 @@ animation = @gif for (i, N) in enumerate(Ns)
 end
 ```
 
-The animation above shows that with increasing evidence our belief about the probability of heads in a coin flip slowly adjusts towards the true value. The orange line in the animation represents the true probability of seeing heads on a single coin flip, while the mode of the distribution shows what the model believes the probability of a heads is given the evidence it has seen.
+The animation above shows that with increasing evidence our belief about the probability of heads in a coin flip slowly adjusts towards the true value.
+The orange line in the animation represents the true probability of seeing heads on a single coin flip, while the mode of the distribution shows what the model believes the probability of a heads is given the evidence it has seen.
+
+For the mathematically inclined, the $\operatorname{Beta}$ distribution is updated by adding each coin flip to the parameters $\alpha$ and $\beta$ of the distribution.
+Initially, the parameters are defined as $\alpha = 1$ and $\beta = 1$.
+Over time, with more and more coin flips, $\alpha$ and $\beta$ will be approximately equal to each other as we are equally likely to flip a heads or a tails.
+
+The mean of the $\operatorname{Beta}(\alpha, \beta)$ distribution is
+
+$$\operatorname{E}[X] = \dfrac{\alpha}{\alpha+\beta}.$$
+
+This implies that the plot of the distribution will become centered around 0.5 for a large enough number of coin flips, as we expect $\alpha \approx \beta$.
+
+The variance of the $\operatorname{Beta}(\alpha, \beta)$ distribution is
+
+$$\operatorname{var}[X] = \dfrac{\alpha\beta}{(\alpha + \beta)^2 (\alpha + \beta + 1)}.$$
+
+Thus the variance of the distribution will approach 0 with more and more samples, as the denominator will grow faster than will the numerator.
+More samples means less variance.
+This implies that the distribution will reflect less uncertainty about the probability of receiving a heads and the plot will become more tightly centered around 0.5 for a large enough number of coin flips.
 
 ### Coin Flipping With Turing
 
-In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior). Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
+In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior).
+Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
```

Comment:
Shouldn't this come before the technical discussion above?

Reply:
```suggestion
In the previous example, we used the fact that our prior distribution is a [conjugate prior](https://en.wikipedia.org/wiki/Conjugate_prior).
Note that a closed-form expression (the `updated_belief` expression) for the posterior is not accessible in general and usually does not exist for more interesting models.
```
```suggestion
```